.gitignore

```diff
@@ -51,4 +51,7 @@ dist/
 
 # build files
 src/versioninfo.json
-src/resource.syso
\ No newline at end of file
+src/resource.syso
+
+# kafka encoded password
+tests/integration/jmxremote/jmxremote.password
```

Comment:
Nit: it was already missing, but I would include the EOL

Reply:
done